moveit_ros/warehouse/src/save_as_text.cpp

```diff
@@ -56,9 +56,8 @@ static const std::string ROBOT_DESCRIPTION = "robot_description";
 typedef std::pair<geometry_msgs::msg::Point, geometry_msgs::msg::Quaternion> LinkConstraintPair;
 typedef std::map<std::string, LinkConstraintPair> LinkConstraintMap;
 
-using moveit::getLogger;
-
-void collectLinkConstraints(const moveit_msgs::msg::Constraints& constraints, LinkConstraintMap& lcmap)
+void collectLinkConstraints(const moveit_msgs::msg::Constraints& constraints, LinkConstraintMap& lcmap,
+                            const rclcpp::Logger& logger)
```

Comment:
Why was this function in particular modified to accept a logger vs. using `moveit::get_logger()`?